### ISSUE TITLE
Warning tooltip issue with Material UI 1.4.0

### ISSUE THREAD

**Issue by jkeruzec** [gregnb/mui-datatables#100]
After update of new Material UI 1.4.0, there is a warning concerning tooltip reimplentation : 

```js
Warning: Failed prop type: The following properties are not supported: `className`. Please remove them.
    in Tooltip (created by WithStyles(Tooltip))
    in WithStyles(Tooltip) (created by l)
    in th (created by TableCell)
    in TableCell (created by WithStyles(TableCell))
    in WithStyles(TableCell) (created by l)
    in l (created by WithStyles(l))
    in WithStyles(l) (created by l)
    in tr (created by TableRow)
    in TableRow (created by WithStyles(TableRow))
    in WithStyles(TableRow) (created by t)
    in t (created by WithStyles(t))
    in WithStyles(t) (created by l)
    in thead (created by TableHead)
    in TableHead (created by WithStyles(TableHead))
    in WithStyles(TableHead) (created by l)
    in l (created by WithStyles(l))
    in WithStyles(l) (created by t)
    in table (created by Table)
    in Table (created by WithStyles(Table))
    in WithStyles(Table) (created by t)
    in div (created by t)
    in div (created by Paper)
    in Paper (created by WithStyles(Paper))
    in WithStyles(Paper) (created by t)
    in t (created by WithStyles(t))
```

**Comment by Kalmuraee** [comment]
I have the same Issue here

**Comment by gregnb** [comment]
Just published 2.0.0-beta-17 and this should be addressed
